README.md

```diff
@@ -1,3 +1,52 @@
-In order to run this project locally, please create .env.test and .env.development. files. In each file you should specify the database which you can find in /db/setup.sql
+The big News app - https://the-big-news-app.onrender.com/
 
-For example, the .env.development should contain the following PGDATABASE=nc_news;
+This project is a news application which works similar to the popular Platform Reddit and provides users with the ability to create, search update and delete articles.
+
+Some of the key features include:
+
+- Searching existing articles by article id or filtering by topic
+- A voting system to ‘score’ articles from users
+- Posting comments
+
+Instructions for use:
+
+Please note this project is run with node js and PQSL
+
+Your node.js version should be compatible with the current version used by express and PG. Node-postgres is compatible with node 8.x, 10.x, 12.x and 14.x To use node >= 14.x you will need to install pg@8.2.x
```

Comment:
I wouldn't add this as the 'current' version could refer to future versions very quickly so this won't age well. You only need to say minimum requirements and set these as whatever you're current versions are right now :)